tests/scenarios/scenario1/pool-clear/3.check-node-pool.test.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+const testUtils = require('../../../../lib/utils/test-utils')
+const utils = require('./utils')
+
+describe('New wallet which was sent transaction', () => {
+  it('should have coins in the wallet', async () => {
+    const response = await testUtils.request('GET', 'transactions/unconfirmed')
+    testUtils.expectSuccessful(response)
+    const transactions = response.data.data
+    
+    expect(transactions.length).toBe(0) // transaction was removed from pool
```

Comment:
How do we ensure here that the transaction was indeed forged by other nodes before we arrive here? I think we should have an intermediate test script which polls the other node(s) and waits for the transaction to be confirmed with them before starting our node1.

Reply:
The transaction is broadcast to the other nodes, so it gets forged in the next block or so. In the last version I'm waiting around 10 blocks so the tx should be confirmed. (if not there is an issue and it is good that the test fail anyway)